test/dynamo/test_activation_checkpointing.py

```diff
@@ -1000,6 +1002,98 @@ def fn(x, y):
         ):
             self._validate(fn, backend, x, y)
 
+    @torch._dynamo.config.patch(inline_inbuilt_nn_modules=True)
+    def test_compile_selective_checkpoint_reparameterization(self):
```

Comment:
maybe a nit: shall we call it parametrization rather than reparametrization in this file?
e.g. I didn't see any word reparametrization from https://pytorch.org/tutorials/intermediate/parametrizations.html

Reply:
Good catch! renamed it to just `parametrization`